### ISSUE TITLE
Invalid tag format?

### ISSUE THREAD

**Issue by callmeradical** [harbur/captain#63]
During tagging after a build
xxxxxxxxxxxxx.dkr.ecr.us-west-2.amazonaws.com/ss:tags.1.0.1^0 <- no idea how this got there
API error (500): {"message":"invalid tag format"}

**Comment by callmeradical** [comment]
I was able to reproduce this by creating a new repo

```bash
$ git init
$ touch README.md
$ git add README.md
$ git commit -am 'Testing'
$ git tag -a 1.0.0
$ captain build


When you issue the command that captain is looking at for branches
```bash
$git name-rev --name-only HEAD
```
You get 
```bash
tags/1.0.0^0
```

Seems unusual, but I added a small fix to get rid of this. Seems to work fine.

**Comment by Jotschi** [comment]
I just encountered the same error. Is that already fixed?

**Comment by berni2288** [comment]
Same error here.

**Comment by spiddy** [comment]
fixed by #84
